admin_guide/persistent_storage_nfs.adoc

```diff
@@ -84,3 +84,18 @@ reclamation of discarded resources can be configured for each persistent volume.
 Many ways to script these tasks exist.  We've created an link:https://github.com/openshift/openshift-ansible/tree/master/roles/kube_nfs_volumes[Ansible playbook] to help you get started.
 
 
+== SELinux and NFS export settings
+
+By default, SELinux will disallow a connection from a pod to a remote NFS server.  Enable this in SELinux on each node with the following command:
+
+```
+setsebool virt_use_nfs 1
+```
+
+Additionally, each exported volume on the NFS server itself must conform to the following:
+
+* each export must be: ```/example_fs *(rw,no_root_squash)```
+* each export is owned by nfsnobody: ```chmod nfsnobody:nfsnobody```
+* each export must have the permissions: ```chmod 700```
+
+Disabling SELinux entirely with ```setenforce 0``` would also work but is generally discouraged.
```

Comment:
Wouldn't even mention this. It's as easy to copy/paste `setsebool virt_use_nfs 1` as this.